Tests fail to find the module ewah_bool_utils.ewah_bool_wrap

```
===>  Testing for py39-ewah-bool-utils-1.0.2
===>   py39-ewah-bool-utils-1.0.2 depends on package: py39-pytest>=7,1 - found
===>   py39-ewah-bool-utils-1.0.2 depends on file: /usr/local/bin/python3.9 - found
cd /usr/ports/devel/py-ewah-bool-utils/work-py39/ewah_bool_utils-1.0.2/tests && /usr/bin/env XDG_DATA_HOME=/usr/ports/devel/py-ewah-bool-utils/work-py39  XDG_CONFIG_HOME=/usr/ports/devel/py-ewah-bool-utils/work-py39  XDG_CACHE_HOME=/usr/ports/devel/py-ewah-bool-utils/work-py39/.cache  HOME=/usr/ports/devel/py-ewah-bool-utils/work-py39 PATH=/usr/local/libexec/ccache:/usr/ports/devel/py-ewah-bool-utils/work-py39/.bin:/home/yuri/bin:/sbin:/bin:/usr/sbin:/usr/bin:/usr/local/sbin:/usr/local/bin PKG_CONFIG_LIBDIR=/usr/ports/devel/py-ewah-bool-utils/work-py39/.pkgconfig:/usr/local/libdata/pkgconfig:/usr/local/share/pkgconfig:/usr/libdata/pkgconfig MK_DEBUG_FILES=no MK_KERNEL_SYMBOLS=no SHELL=/bin/sh NO_LINT=YES PREFIX=/usr/local  LOCALBASE=/usr/local  CC="cc" CFLAGS="-O2 -pipe  -fstack-protector-strong -fno-strict-aliasing "  CPP="cpp" CPPFLAGS=""  LDFLAGS=" -fstack-protector-strong " LIBS=""  CXX="c++" CXXFLAGS="-O2 -pipe -fstack-protector-strong -fno-strict-aliasing  "  MANPREFIX="/usr/local" CCACHE_DIR="/tmp/.ccache" BSD_INSTALL_PROGRAM="install  -s -m 555"  BSD_INSTALL_LIB="install  -s -m 0644"  BSD_INSTALL_SCRIPT="install  -m 555"  BSD_INSTALL_DATA="install  -m 0644"  BSD_INSTALL_MAN="install  -m 444" PYTHONPATH=/usr/ports/devel/py-ewah-bool-utils/work-py39/stage/usr/local/lib/python3.9/site-packages /usr/local/bin/python3.9 -m pytest -k '' -rs -v -o addopts= 
============================================================================================================ test session starts =============================================================================================================
platform freebsd13 -- Python 3.9.16, pytest-7.3.1, pluggy-1.0.0 -- /usr/local/bin/python3.9
cachedir: .pytest_cache
hypothesis profile 'default' -> database=DirectoryBasedExampleDatabase('/usr/ports/devel/py-ewah-bool-utils/work-py39/ewah_bool_utils-1.0.2/tests/.hypothesis/examples')
Using --randomly-seed=174909104
rootdir: /usr/ports/devel/py-ewah-bool-utils/work-py39/ewah_bool_utils-1.0.2
plugins: forked-1.6.0, hypothesis-6.72.4, cov-2.9.0, mypy-plugins-1.11.1, randomly-3.12.0, timeout-2.1.0, flaky-3.7.0, rerunfailures-11.1.2, xdist-2.5.0, env-0.6.2, mock-3.10.0
collected 0 items / 1 error                                                                                                                                                                                                                  

=================================================================================================================== ERRORS ===================================================================================================================
_______________________________________________________________________________________________ ERROR collecting tests/test_ewah_bool_utils.py _______________________________________________________________________________________________
ImportError while importing test module '/usr/ports/devel/py-ewah-bool-utils/work-py39/ewah_bool_utils-1.0.2/tests/test_ewah_bool_utils.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/usr/local/lib/python3.9/importlib/__init__.py:127: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
test_ewah_bool_utils.py:7: in <module>
    from ewah_bool_utils._testing import (
../ewah_bool_utils/__init__.py:4: in <module>
    from .ewah_bool_wrap import *
E   ModuleNotFoundError: No module named 'ewah_bool_utils.ewah_bool_wrap'
!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
============================================================================================================== 1 error in 0.45s ==============================================================================================================
```

Version: 1.0.2
Python-3.9
FreeBSD 13.2

--- a/ewah_bool_utils/__init__.py
+++ b/ewah_bool_utils/__init__.py
@@ -1,7 +1,7 @@
 """Top-level package for EWAH Bool Utils."""
 
 
-from .ewah_bool_wrap import *
+from ewah_bool_utils.ewah_bool_wrap import *
 
 
 def get_include():

